event/class.go

```diff
@@ -42,12 +46,13 @@ type Class struct {
 }
 
 // A Example is a real query and its database, timestamp, and Query_time.
-// If the query is larger than MAX_EXAMPLE_BYTES, it is truncated and "..."
+// If the query is larger than MaxExampleBytes, it is truncated and TruncatedExampleSuffix
 // is appended.
 type Example struct {
 	QueryTime float64 // Query_time
 	Db        string  // Schema: <db> or USE <db>
-	Query     string  // truncated to MAX_EXAMPLE_BYTES
+	Query     string  // truncated to MaxExampleBytes
+	Size      int     `json:",omitempty"` // Original size of query.
```

Comment:
Maybe "OriginalLength"? "Size" sounds like a number of returned rows or something, at least for me.

Reply:
`OriginalQueryLength`? :) Well, it's example so the size of example, not number of returned rows... but I get your point.
I was thinking about `Size`, `Length`, I've ignored longer names as a) go suggest short names b) it's in json, so takes more space (sorry, no binary format) c) same for db schema, it just doesn't look good `original_query_length` in sql, considering other columns have half the size.

In any case, fine for me, just let me know which name you pick so I could update all PRs, because once we released that it will be harder to change because of schema.

Reply:
Ok, I don't really mind. If "Size" is consistently used everywhere, so be it.